docs/developer/testing/TESTING.md

```diff
@@ -20,6 +20,7 @@ To get started, you need to provide the following **required** environment varia
 | `VSL_REGION` | The region of snapshotLocations | - | true |
 | `BSL_REGION` | The region of backupLocations | `us-east-1` | false |
 | `OADP_TEST_NAMESPACE` | The namespace where OADP operator is installed | `openshift-adp` | false |
+| `OPENSHIFT_CI` | Disable colored output from tests suite run | `true` | false |
```

Comment:
It would be helpful to change this name to something that matches what it actually does, "OPENSHIFT_CI" sounds kind of vague. The name would make more sense if there were other options that it controlled, to indicate the tests are running in some kind of "OpenShift-CI mode". But if it only does this one thing, it will be less confusing to call it something else.

Reply:
The only other place it is used is here https://github.com/openshift/oadp-operator/blob/master/tests/e2e/scripts/azure_settings.sh#L9-L11(did not use azure yet, I do not understand what this does)

I can create a separate environment variable for color